includes/wp-admin/table/class-blocked-actors.php

```diff
@@ -397,6 +397,19 @@ protected function handle_row_actions( $item, $column_name, $primary ) {
 			),
 		);
 
+		/**
+		 * Filters the array of row action links on the Blocked Actors list table.
+		 *
+		 * This filter is evaluated for each blocked actor item in the list table.
+		 *
+		 * @since unreleased
+		 *
+		 * @param string[] $actions An array of row action links. Defaults are
+		 *                          'Unblock'.
+		 * @param array    $item    The current blocked actor item.
+		 */
+		$actions = apply_filters( 'activitypub_blocked_actors_row_actions', $actions, $item );
```

Comment:
We could also call `parent::handle_row_actions` but then we would have to unhook all of the default `row_action` links.